extensions/MESA/MESA_framebuffer_flip_x.txt

```diff
@@ -0,0 +1,91 @@
+Name
+
+    MESA_framebuffer_flip_x
+
+Name Strings
+
+    GL_MESA_framebuffer_flip_x
+
+Contact
+
+    Cici Ruan <ruanc@chromium.org>
+
+Contributors
+
+    Cici Ruan, Google
+    Kristian Kristensen‎, Google
+    Fritz Koenig, Google
+    Rob Clark, Google
+    Chad Versace, Google
+
+Status
+
+    Proposal
+
+Version
+
+    Last Modified Date: April 8, 2020
+    Revision: 1
+
+Number
+
+    OpenGL ES Extension 327
```

Comment:
Since these extensions are given both GL and ES extension numbers, those should both be reflected here as well.

Reply:
Added. Thanks for pointing it out.